packages/smooth_app/lib/pages/product/add_basic_details_page.dart

```diff
@@ -151,6 +152,10 @@ class _AddBasicDetailsPageState extends State<AddBasicDetailsPage> {
                     if (!_formKey.currentState!.validate()) {
                       return;
                     }
+                    AnalyticsHelper.trackProductEdit(
+                        AnalyticsEditEvents.basicDetials,
+                        _product.barcode!,
+                        true);
```

Comment:
For the nice dart formatting 😉 

```suggestion
                        true,);
```